packages/shared/src/hooks/useCopySquadInvitation.ts

```diff
@@ -0,0 +1,29 @@
+import { CopyNotifyFunction, useCopyLink } from './useCopyLink';
+import { useAnalyticsContext } from '../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../lib/analytics';
+import { Squad } from '../graphql/squads';
+
+interface UseCopySquadInvitationProps {
+  squad: Squad;
+  origin?: string;
+}
+
+export const useCopySquadInvitation = ({
```

Comment:
Just make sure this is used everywhere we currently use this flow and origin is always set.

Reply:
Right. Let me make the origin as required.